Applications/Utils/MeshEdit/queryMesh.cpp

```diff
@@ -63,6 +61,12 @@ int main(int argc, char *argv[])
     }
     selected_node_ids.insert(selected_node_ids.end(), nodeId_arg.getValue().begin(), nodeId_arg.getValue().end());
 
+    MeshLib::PropertyVector<int>* materialIds(nullptr);
```

Comment:
I was close to suggest it too, especially because of `const` but the readability degrades IMO.